### ISSUE TITLE
restitution of FixtureDef

### ISSUE THREAD

**Issue by DavidChZh** [flame-engine/flame#3136]
### What happened?

https://github.com/flame-engine/flame/assets/24714777/00a25af9-1dce-4cc4-82ba-8ff2e6881f0f
In this video, I created some polygons through the BodyComponent. 
The linearDamping and angularDamping of the BodyDef attribute are both set to 1, and the friction of FixtureDef is also set to 1, leaving only the retention of 0.3. 
In this case, objects will always undergo slight friction jitter.
I tried to set the retention to 0, which will not occur, but I cannot do so because my sphere needs to be elastic. 
What should I do in this situation?

### What do you expect?

When the resolution attribute is set to non-0, objects are stable enough

### Execute in a terminal and put output into the code block below

Output of: flutter doctor -v

### Affected platforms

Android

### Are you interested in working on a PR for this?

- [ ] I want to work on this

**Comment by spydon** [comment]
I guess you mean restitution in your text, and not retention?
Have you tried to set the friction to something close to 1.0, like 0.999?

**Comment by DavidChZh** [comment]
> I guess you mean restitution in your text, and not retention? Have you tried to set the friction to something close to 1.0, like 0.999?

Yes, it is resolution.
I haven't tried before. I just tried to set the frequency to 0.999. The effect is the same as before

**Comment by DavidChZh** [comment]
https://forum.cocos.org/t/box2d/85979
Based on this scheme, when the number of impacts of the same ball exceeds a certain threshold, the elasticity of the ball is dynamically set to 0

**Comment by DavidChZh** [comment]
@spydon Thank you for your reply

**Comment by DavidChZh** [comment]
I found a more suitable parameter configuration. I lowered the gravity of the game world so that the ball no longer trembles. At the same time, I increased the falling speed of the ball by configuring the gravityScale parameter, and restored the gravityScale configuration at the moment of landing so that the ball can regain its elasticity. Now they look smoother.
@spydon  Sorry to @ you again, but I wanted to share my joy with you. More importantly, if there are similar questions later, I hope this helps.
